Example/Glia Sample Integration/Presenters/ContentViewPresenter.swift

```diff
@@ -0,0 +1,115 @@
+import SwiftUI
+import SalemoveSDK
+import GliaWidgets
+
+class ContentViewPresenter: ObservableObject {
+    private let gliaConfiguration: GliaConfiguration
+    private var theme = Theme()
+    private var queue: SalemoveSDK.Queue?
+
+    @Published var isVideoAvailable: Bool = false
```

Comment:
Why do you need `@Published` attribute?

Reply:
A view that needs to be rerendered when this value changes, can't do it without @Published property wrapper.

Reply:
So `ObservableObject` invalidates View from the box when `@Published` properties are changed and you don't need to subscribe explicitly? 
Something like 
```swift
presenter.$isVideoAvailable
    .sink() {
        reloadViewState()
}
```

Reply:
What is the benefit of it?